server.go

```diff
@@ -15,6 +15,16 @@ import (
 // PORT defines where we serve the backend.
 const PORT = ":8081"
 
+func applyMiddlewareCreator(middlewares ...(func(func(w http.ResponseWriter, r *http.Request)) func(w http.ResponseWriter, r *http.Request))) func(func(http.ResponseWriter, *http.Request)) func(http.ResponseWriter, *http.Request) {
```

Comment:
I require help on how to make this method signature readable

Reply:
This function, in my opinion, is not needed for our current purposes. While I appreciate the thought of including a function like this for future use, the execution could be smoother and would probably be something we include in our `middleware/` folder. This said, input from the rest of the team on the matter may be desirable.